src/library/simplified/extension/city-name.js

```diff
@@ -47,6 +47,10 @@ else if(utag.isFSR() && b["entity.flightSearch.searchParameters.arrivalAirportCi
         b["city"] = cityState[0];
     }
 }
+else if((utag.isRailSearchResults) && b["qp.destinationName"] != undefined) 
+{
+    b["city"] = b["qp.destinationName"].split(",")[0];
```

Comment:
Currently, we dont have "entity.railSearch.searchParameters.arrivalStation.name" correctly populated. It has only station name. From this parameter, we cannot retrieve city, state, country name.
![image](https://cloud.githubusercontent.com/assets/21168911/24895634/112e23aa-1eaf-11e7-8400-ace39d7a34c4.png)